src/components/CommentCard.jsx

```diff
@@ -4,30 +4,65 @@ import CardHeader from '@mui/material/CardHeader';
 import Avatar from '@mui/material/Avatar';
 import { red } from '@mui/material/colors';
 import { Typography } from '@mui/material';
+import DeleteOutlinedIcon from '@mui/icons-material/DeleteOutlined';
+import IconButton from '@mui/material/IconButton';
+import { useContext } from 'react';
+import { UserContext } from './contexts';
+import { deleteComment } from '../network/network';
 
-
-
-export default function CommentCard({comment, users}){
+export default function CommentCard({comment, users, comments, setComments}){
     
-    const thisUser = users.filter((user)=>user.username===comment.author)
+    const thisUser = users.find((user)=>user.username===comment.author)
     const date = new Date(comment.created_at) 
-
+    const {user} = useContext(UserContext)
+    
+    
+    const handleDeleteComment = async ()=>{
+        const res = await deleteComment(comment.comment_id)
+        
+        if (await res.status === 204){
+            const index = comments.findIndex((item)=>comment.comment_id===item.comment_id)
+            comments.splice(index,1)
+            console.log(comments)
+        
+            
+            setComments([...comments])
+        }
+    }
+    
+    const CommentBin = ()=>{
+        return (
+            <IconButton
+            type="button" 
+                // sx={{ marginLeft: '80', p: '10px' }} 
```

Comment:
Try to avoid leaving in unused code

Reply:
Left here as a reminder to adjust layout - will be removed once that job is done.